src/TurbulenceClosures/turbulence_closure_implementations/ri_based_vertical_diffusivity.jl

```diff
@@ -198,7 +194,7 @@ end
     κᵉ = ifelse(entraining, Cᵉ, zero(grid))
 
     # Shear mixing diffusivity and viscosity (diffused in the horizontal to add non-locality)
-    Ri = ℑxyᶜᶜᵃ(i, j, k, grid, ℑxyᶠᶠᵃ, Riᶜᶜᶠ, velocities, tracers, buoyancy)
+    Ri = Riᶜᶜᶠ(i, j, k, grid, velocities, N²)
```

Comment:
it's elsewhere defined with `tracers, buoyancy` so might be better to keep here. Should all be inlined anyways so no difference in compute.